riskified-sdk/src/main/java/com/riskified/validations/Validate.java

```diff
@@ -0,0 +1,63 @@
+package com.riskified.validations;
+
+import org.apache.commons.validator.routines.EmailValidator;
+import org.apache.commons.validator.routines.InetAddressValidator;
+
+public class Validate {
+
+	
+	static public void notNull(Object obj, String fieldName) throws FieldBadFormatException {
+		if(obj == null) {
+			throw new FieldBadFormatException(fieldName + " can't be null.");
+		}
+	}
+	
+	static public void stringNotNullOrEmpty(String str, String fieldName) throws FieldBadFormatException {
```

Comment:
perhaps it would be cleaner to pass the object (this) and field name to avoid duplicating arguments? ie, change all from:
`Validate.stringNotNullOrEmpty(this.title, "Title");`
to:
`Validate.stringNotNullOrEmpty(this, "title");`